test/ingestors/highlight.test.ts

```diff
@@ -45,40 +52,39 @@ describe('highlight', function () {
 
   it('createMintTemplateForUrl: Returns a mint template for a supported URL', async function () {
     const ingestor = new HighlightIngestor();
-    const url = 'https://highlight.xyz/mint/665fa33f07b3436991e55632';
+    const url = 'https://highlight.xyz/mint/66856628ff8a01fdccc132f4';
```

Comment:
This URL for whatever reason no longer returns all data.

I suspect this may come up again in the future when others age out... favoring a long-running one for now